OKR/PROJECT.md

```diff
@@ -2,8 +2,28 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
-## 2018 Q4
+## 2019 Q1
 
+- **The IPFS Org gets a fully dedicated Package Managers Working Group**
+  - `PX` - @??? - A Package Manager Working Group Roadmap is created
+  - `PX` - @??? - Onboard 2+ contributors to this Working Group       
+- **Support and guide the IPFSConf planning and Content Curation**
+  - `PX` - @daviddias - The IPFSConf is scheduled, announced and there is a CFP published   
+  - `PX` - @??? - Working Groups have their goals established for IPFSConf    
+- **The IPFS Working Groups feel supported and they have the information they need to make the best decisions**
+  - `PX` - @??? - Ship the first IPFS User Quarterly Report. It documents achievements and demos of new capabilities
+  - `PX` - @??? - We gather, process and present metrics for the IPFS Project    
+  - `PX` - @??? - We grow the Project WG to have a team capable of answering to all IPFS Working Group needs.
+  - `PX` - @??? - Working group roadmaps are solidified, presented, and used to chart Q1 and Q2 planning
+- **The Working Groups are widely known in the IPFS Community and easy to get onboarded to**
```

Comment:
Talked with Arkadiy and we want to add some collaborations OKRs here - so maybe just make this: 

```suggestion
- **The IPFS Community is supportive, responsive, and easy to get onboarded to**
```

KR: Biweekly updates published to IPFS working groups highlighting patterns in top developer needs
KR: Ship strategy document outlining tooling, pipeline, and management for IPFS collabs
KR: 10 collab engagements result in valuable output for the wider community (feature, blog post, collaborative meeting, bug fix, etc) 
KR: There are zero dropped/mishandled collabs